content/docs/source-git/work-with-source-git/pull-upstream-fixes.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "Pull fixes from the upstream"
+date: 2021-01-26T23:59:59+01:00
+draft: false
+weight: 2
+---
+
+# Pull fixes from the upstream
+
+One of the common tasks of being a downstream maintainer is to pull fixes from
+the upstream codebase. This starts by creating a new bugzilla with an issue in
+the distribution which requires a code-change in the affected package. The
+maintainer needs to pull it from upstream when:
+1. The fix is not released yet by the upstream - this implies pulling the fix
+   from the main development upstream branch.
+2. If the fix is already released, one can either update to that upstream
+   release or only pull commits with the fix. Please consult [Fedora Packaging
+   Guidelines](https://docs.fedoraproject.org/en-US/packaging-guidelines/) if
+   you are unsure how to proceed.
+
+
+## Adding changes
+
+Since your source-git repository shares git history with upstream, you can
+easily cherry-pick commits which you want to have in the downstream and packit
+is then able to turn those commits into patch files with the ability to
+configure the whole process.
+
+As an example, let's try to do this with systemd, in the example below we'd be
+using https://gitlab.com/packit-service/src/systemd.
+```
+$ git remote -v
+origin      git@gitlab.com:TomasTomecek/systemd-stable.git (push)
+source-git  git@gitlab.com:packit-service/src/systemd.git (push)
+upstream    git@github.com:systemd/systemd-stable.git (push)
+```
+
+We have 3 remotes in the repo:
+1. `origin` — a fork of the source-git repo
```

Comment:
Fork-remotes set up by `fedpkg fork` are called by the FAS username, not `origin`. Should we also follow this pattern in source-git?

Reply:
sounds good! this was my random local setup when creating some source-git repo, will do!

also, I never used `fedpkg fork`